single_tech_samples/synapseanalytics/sample2_synapseserverless/synapseartifacts/notebooks/Nb_NYCTaxi_Config_Operations_Library.ipynb

```diff
@@ -3,185 +3,57 @@
     {
       "attachments": {},
       "cell_type": "markdown",
+      "source": [
+        "## Install Required Python Modules\n"
+      ],
       "metadata": {
         "nteract": {
           "transient": {
             "deleting": false
           }
         }
-      },
-      "source": [
-        "## Install Required Python Modules\n"
-      ]
+      }
     },
     {
       "cell_type": "markdown",
-      "metadata": {
-        "nteract": {
-          "transient": {
-            "deleting": false
-          }
-        }
-      },
       "source": [
-        "*This module installation can be moved to package installation using \"requirements.txt\"*\n",
         "\n",
-        "*`az synapse spark pool update` with --library-requirements requirements.txt*\n",
+        "*`az synapse spark pool update` with --library-requirements requirements.txt* - *Concern: Long time(~ 20 mins) for the statement to return status*\n",
```

Comment:
should we remove these internal comments/surface them in the readme instead?

Reply:
They are also included in the comparison table in the ReadMe file. I think doesn't harm to have them in the notebook as a reminder.